graylog2-rest-client/src/main/java/org/graylog2/restclient/models/api/requests/dashboards/WidgetPositionRequest.java

```diff
@@ -16,13 +16,12 @@
  */
 package org.graylog2.restclient.models.api.requests.dashboards;
 
-/**
- * @author Lennart Koopmann <lennart@torch.sh>
- */
 public class WidgetPositionRequest {
```

Comment:
I guess you mean to consolidate it with `org.graylog2.rest.resources.dashboards.requests.WidgetPositions`. I will take care of that.